CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+### 0.10.0 (unreleased)
+
+- Breaking changes
+  - Auto derivation of readers and writers for case classes and sealed traits is now disabled by default. Now users need
+    to import `pureconfig.generic.auto._` everywhere a config is loaded or written (e.g. in files with calls to
+    `loadConfig`);
+  - The `AllowMissingKeys` trait was renamed to `ReadsMissingKeys`;
+  - Dropped support for Scala 2.10.
```

Comment:
We should also mention here that the primitive readers for `Boolean`, `Double`, `Float`, `Int`, `Long` and `Short` no longer return the `EmptyStringFound` in the presence of empty strings, returning `WrongType` instead.

Reply:
Do you categorize that as a breaking change? I don't think it was actually documented anywhere and we already changed many times before the errors that we provided without informing users. As such, I'm not sure where to put it in the CHANGELOG, if at all.

Reply:
I think it is a breaking change, but it might not be above the threshold of breaking changes we want to report in the CHANGELOG. If there was no documented or expected behavior for those readers, then I think it's ok to not include it.